package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "awsx",
   "description": "AWS CLI profile switcher with MFA support",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
I think Ian's plan is to bump the version once all the hackathon changes are in and release. I might be wrong though, @iansu ?

Reply:
Yeah, I think it makes sense to batch them together.